scripts/webrtcclientsdk.js

```diff
@@ -82,7 +84,7 @@ define(function (require) {
 
     BJN.RTCManager.setBandwidth(options.bandWidth);
     MediaStarted = false;
-    startLocalStream();
+    startLocalStream(true);
```

Comment:
Alternative: pass in a `config` object.
E.g., startLocalStream({ initializing: true });
Make it easier to see what the argument is used for.